### ISSUE TITLE
Test suite fails with import errors

### ISSUE THREAD

**Issue by TheChymera** [wxWidgets/Phoenix#1025]
**Operating system**: Gentoo Linux
**wxPython version & source**: 4.0.3 from https://files.pythonhosted.org/packages/dd/31/bd55ab40e406a026a7fda0bb5eb61f466682544ae91ac26267c750f5e618/wxPython-4.0.3.tar.gz
**Python version & source**: 2.7.15 from distro

**Description of the problem**:

The software builds without error but fails to pass the test suite (see [log](http://bpaste.net/show/5c1fdd3ead39)). Here's the fail snippet for your convenience:

```
E.........EEEE....EEE..
======================================================================
ERROR: test_interpreter (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_interpreter
Traceback (most recent call last):
  File "/usr/lib64/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib64/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "test_interpreter.py", line 11, in <module>
    import interpreter
  File "../interpreter.py", line 10, in <module>
    from . import dispatcher
ValueError: Attempted relative import in non-package


======================================================================
ERROR: test_getAutoCompleteList (test_introspect.GetAutoCompleteListTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_introspect.py", line 749, in test_getAutoCompleteList
    result = introspect.getAutoCompleteList(item)
  File "../introspect.py", line 20, in getAutoCompleteList
    root = getRoot(command, terminator='.')
  File "../introspect.py", line 225, in getRoot
    if command.startswith(sys.ps2):
AttributeError: 'module' object has no attribute 'ps2'

======================================================================
ERROR: test_getAutoCompleteList_NoDouble (test_introspect.GetAutoCompleteListTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_introspect.py", line 767, in test_getAutoCompleteList_NoDouble
    result = introspect.getAutoCompleteList(item, includeDouble=0)
  File "../introspect.py", line 20, in getAutoCompleteList
    root = getRoot(command, terminator='.')
  File "../introspect.py", line 225, in getRoot
    if command.startswith(sys.ps2):
AttributeError: 'module' object has no attribute 'ps2'

======================================================================
ERROR: test_getAutoCompleteList_NoSingle (test_introspect.GetAutoCompleteListTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_introspect.py", line 759, in test_getAutoCompleteList_NoSingle
    result = introspect.getAutoCompleteList(item, includeSingle=0)
  File "../introspect.py", line 20, in getAutoCompleteList
    root = getRoot(command, terminator='.')
  File "../introspect.py", line 225, in getRoot
    if command.startswith(sys.ps2):
AttributeError: 'module' object has no attribute 'ps2'

======================================================================
ERROR: test_getAutoCompleteList_NoSingleOrDouble (test_introspect.GetAutoCompleteListTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_introspect.py", line 776, in test_getAutoCompleteList_NoSingleOrDouble
    includeDouble=0)
  File "../introspect.py", line 20, in getAutoCompleteList
    root = getRoot(command, terminator='.')
  File "../introspect.py", line 225, in getRoot
    if command.startswith(sys.ps2):
AttributeError: 'module' object has no attribute 'ps2'

======================================================================
ERROR: test_getRoot (test_introspect.GetRootTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_introspect.py", line 150, in test_getRoot
    self._checkRoot(input, terminator, output)
  File "test_introspect.py", line 98, in _checkRoot
    root = introspect.getRoot(command=input, terminator=terminator)
  File "../introspect.py", line 225, in getRoot
    if command.startswith(sys.ps2):
AttributeError: 'module' object has no attribute 'ps2'

======================================================================
ERROR: test_getRoot_Advanced (test_introspect.GetRootTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_introspect.py", line 180, in test_getRoot_Advanced
    self._checkRoot(input, terminator, output)
  File "test_introspect.py", line 98, in _checkRoot
    root = introspect.getRoot(command=input, terminator=terminator)
  File "../introspect.py", line 225, in getRoot
    if command.startswith(sys.ps2):
AttributeError: 'module' object has no attribute 'ps2'

======================================================================
ERROR: test_getRoot_EmptyTypes (test_introspect.GetRootTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_introspect.py", line 341, in test_getRoot_EmptyTypes
    self._checkRoot(input, terminator, output)
  File "test_introspect.py", line 98, in _checkRoot
    root = introspect.getRoot(command=input, terminator=terminator)
  File "../introspect.py", line 225, in getRoot
    if command.startswith(sys.ps2):
AttributeError: 'module' object has no attribute 'ps2'

----------------------------------------------------------------------
Ran 23 tests in 0.083s

FAILED (errors=8)
```

**Comment by RobinD42** [comment]
Hmmm... Apparently nobody has updated the pycrust tests since the port to Phoenix, and since all the other tests are in the unittests folder it was easy to miss. PRs are welcome.

**Comment by TheChymera** [comment]
Sadly I can't dig into this any further :(

But just to disable these broken tests for your 4.0.3 release, xfailing `GetAttributeNamesTestCase` and `GetAutoCompleteListTestCase` under `wx/py/tests/test_introspect.py` would be sufficient?

If so do you want a PR with this "fix"?

**Comment by RobinD42** [comment]
sure

**Comment by RobinD42** [comment]
Fixed by #1057
